src/view-utilities.js

```diff
@@ -335,7 +335,116 @@ var viewUtilities = function (cy, options) {
     cy.autounselectify(false);
     marqueeZoomEnabled = false;
   };
+  var geometric = require('geometric')
+
+  instance.enableLassoMode = function ({callback = undefined, lineColour = "#d67614", lineWidth = 3} = {}) {
```

Comment:
Let's do not take line style as function paramater, instead create a new extension option named as `lassoStyle` with current default values: `lassoStyle: {lineColor: "#d67614", lineWidth: 3},`. See options in the index.js file.  
Keep only callback as function parameter, but not inside an object.